akvo/rsr/models/organisation.py

```diff
@@ -456,6 +457,25 @@ def iati_file(self):
 
     # New API
 
+    def has_multiple_project_currencies(self):
```

Comment:
Could be rewritten to:
`return self.published_projects().distinct().count() != self.org_currency_projects_count()`